src/mmap_windows.rs

```diff
@@ -175,6 +175,11 @@ impl MmapRegion {
     pub fn file_offset(&self) -> Option<&FileOffset> {
         self.file_offset.as_ref()
     }
+
+    /// Windows doesn't have hugetlbfs.
```

Comment:
Can we rephrase this a bit? Right now it sounds more like a comment than documentation for the public interface. How about something like: "This information is not available on Windows platforms."?

Reply:
Updated.